src/wasm.rs

```diff
@@ -285,23 +285,23 @@ pub enum Instruction {
     Nop,
     Select,
 
-    GetLocal { index: u32 },
-    SetLocal { index: u32 },
-    TeeLocal { index: u32 },
+    LocalGet { index: u32 },
+    LocalSet { index: u32 },
+    LocalTee { index: u32 },
 
-    GetGlobal { index: u32 },
-    SetGlobal { index: u32 },
+    GlobalGet { index: u32 },
+    GlobalSet { index: u32 },
```

Comment:
FYI one reason I created this abstraction layer was to avoid coupling names. If you think this is clearer, more power to you